### PR TITLE
FLUID-5808: Adding missing Access for All page.

### PR DESCRIPTION
**Pull request by jhung**

### CODE REVIEW

**Comment by acheetham** [comment]
@jhung: The 404 page should probably be customized for this site. The heading is currently "Documentation template" and link text in the body just refers to "Documentation Base"

**Comment by acheetham** [comment]
@jhung on the Accessibility Principles page, the link to LearnerNeedsAndPreferences could use some spaces in the link text

**Comment by acheetham** [comment]
on the Accessibility Principles page, the "Cognitive considerations" link is broken

**Comment by acheetham** [comment]
on the Techniques page, the link to the UIO demo is broken

**Comment by acheetham** [comment]
on the Metadata page, the "Schema.org Accessibility Properties" link is broken

**Comment by acheetham** [comment]
on the Learner Needs and Prefs page, the "Landscape of alternatives..." image is pretty small. You click on it to see the full size image, but the full-size image is still tiny - enough to make it basically useless.

**Comment by acheetham** [comment]
In the footer, the links to the IDRC site and the OCADU site don't open in a new tab the way all the other external links do

**Comment by acheetham** [comment]
on the Metadata page, the "Schema.org Accessibility Properties" link is missing the 'external link' arrow and doesn't open in a separate tab

**Comment by acheetham** [comment]
@jhung: The 404 page  heading is still "Documentation template"

**Comment by jhung** [comment]
Re: "Schema.org Accessibility Properties" link is missing the 'external link' arrow and doesn't open in a separate tab"

That link should be a link to another document in the ILDH, so it shouldn't be an external link. Can you confirm that this link actually does that?

Re: 404 title
This has been updated. I can see the diff showing that the title has been changed. Can you delete your ./out directory and confirm this was fixed?

**Comment by acheetham** [comment]
@jhung:
re schema.org: you're right, sorry.
re 404: I'm still seeing "documentation template" in the header and the browser title. Regenerated, cleared cache, etc.

File: src/documents/Techniques.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Supporting flexible styling can be as simple as using Cascading Style Sheets (CS
 
 
 Floe examples:
-* <a href="http://build.fluidproject.org/infusion/demos/uiOptions/demo.html" rel="nofollow" target="_blank" class="link-external">Learner Options</a> (sometimes known as UI Options)
+* <a href="http://build.fluidproject.org/infusion/demos/prefsFramework/" rel="nofollow" target="_blank" class="link-external">Learner Options</a> (sometimes known as UI Options)
```
Comment (acheetham) [inline src/documents/Techniques.md]: I wonder if this should actually point to the uiOptions demo, and not to the prefs framework demo...

**Pull request merged**